packages/as-types/assembly/safeMath.ts

```diff
@@ -40,9 +79,18 @@ export class SafeMath {
   }
 
   /**
-   *
-   * @returns Returns the integer division of two unsigned integers. Reverts on
+   * Safely divides two unsigned 64-bit integers (u64), reverting on
    * division by zero. The result is rounded towards zero.
+   *
+   * @remarks
+   * This function is used to safely divide two unsigned 64-bit integers without causing a division by zero error.
```

Comment:
You should add a remarks that nothing is done to check lose in precision, the rest is ignored.
For instance, what happens if you do 4/3